client/src/main/java/org/eclipse/hono/client/impl/DownstreamSenderFactoryImpl.java

```diff
@@ -65,7 +69,7 @@ protected void onDisconnect() {
                 .compose(v -> connection.executeOnContext(result -> {
                     clientFactory.getOrCreateClient(
                             AddressHelper.getTargetAddress(TelemetryConstants.TELEMETRY_ENDPOINT, tenantId, null, connection.getConfig()),
-                            () -> TelemetrySenderImpl.create(connection, tenantId,
+                            () -> TelemetrySenderImpl.create(connection, tenantId, samplerFactory.create("telemetry"),
```

Comment:
how about using *TelemetryConstants.TELEMETRY_ENDPOINT* instead of a literal string?